fleet/s/dev/c/ruka/observability/loki/values.yaml

```diff
@@ -0,0 +1,52 @@
+# loki/values.yaml
+config:
```

Comment:
The only config I see in the values.yaml is `loki.confg`. https://github.com/grafana/loki/blob/main/production/helm/loki/values.yaml#L127